packages/eslint-config-loris/react.js

```diff
@@ -0,0 +1,80 @@
+module.exports = {
+    plugins: ['react'],
+
+    parserOptions: {
+        ecmaFeatures: {
+            jsx: true
+        }
+    },
+
+    rules: {
+        // http://eslint.org/docs/rules/#stylistic-issues
+        'jsx-quotes': ['error', 'prefer-double'],
+
+        // https://github.com/yannickcr/eslint-plugin-react#list-of-supported-rules
+        'react/default-props-match-prop-types': 'off',
+        'react/display-name': 'error',
```

Comment:
Ok, I understand your point and agree with you. So this rule will be turned off.